### PR TITLE
LNX-185 Server generates DropArea

### PR DESCRIPTION
**Pull request by filip-szweda**
- https://github.com/group-project-gut/lynx-frontend-godot/pull/41
- https://github.com/group-project-gut/lynx-python-common/pull/27
obraz

### CODE REVIEW

**Review by Krystian030 — APPROVED** [review]

File: src/app.py [hunk 1]
```diff
@@ -228,18 +228,21 @@ async def create_tick():
...
 @app.get("/")
-async def get(tick_number: int) -> Dict[str, Union[int, str, List[List[str]]]]:
+async def get(tick_number: int, player: str = -1) -> Dict[str, Union[int, str, List[List[str]]]]:
     logger.debug(f"Starting to get state for tick number {tick_number}")
     state.available_ticks_without_get = int(config["REQUESTS"]["DEFAULT_TICKS_WITHOUT_GET"])
-    # if player has no scene or player tick number is incorrect
-    # TODO remove tick number and use scene hash instead e.g. if player_scene_hash not in self.states.keys():
     current_time_ms = int(time.time() * 1000)
+
+    if tick_number == 0:
```
Comment (EasyJB) [inline src/app.py]: Shouldn't it be -1? As first thing which Godot sends is get wiith -1 tick. But then we need to check if scene was already created.

**Review by EasyJB — COMMENTED** [review]
Also imo we can add, CONST for user, so he can write like goto(BASE), and BASE will be value of position of DropArea.

File: src/app.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ class GlobalState:
     tick_number: int = 0
     last_tick_time: float = 0.0
     available_ticks_without_get: int = 0
+    players: List[str] = field(default_factory=list)
```
Comment (filip-szweda) [inline src/app.py]: Move to `Scene`

File: src/app.py [hunk 3]
```diff
@@ -314,7 +339,8 @@ async def generate_scene():
         url = 'https://scene-generator.kubernetes.blazej-smorawski.com/get_scene'
 
     await clear()
-    response = await post(url, payload={"seed": "test", "width": int(config["REQUESTS"]["MAP_SIZE_X"]), "height": int(config["REQUESTS"]["MAP_SIZE_y"])})
+    response = await post(url,
```
Comment (filip-szweda) [inline src/app.py]: Move in one line

**Comment by filip-szweda** [comment]
> Also imo we can add, CONST for user, so he can write like goto(BASE), and BASE will be value of position of DropArea.

Player can add this const themself, it would be very easy with our tooltips.

**Review by Krystian030 — APPROVED** [review]

**Pull request merged**